rules/rbe_repo.bzl

```diff
@@ -692,6 +711,35 @@ def _expand_outputs(ctx, bazel_version, project_root):
                 result = ctx.execute(args)
                 _print_exec_results("copy %s repo files" % repo, result, True, args)
 
+# Copies all contents of the external repo to a test directory,
+# modifies name of all BUILD files (to enable file_test to operate on them), and
+# creates a root BUILD file in test directory with a filegroup that contains
+# all files.
+def _copy_to_test_dir(ctx):
+    # Copy all files with rsync
+    args = ["rsync", "-aR", "./", "./test"]
+    result = ctx.execute(args)
+    _print_exec_results("copy test output files", result, True, args)
+
+    # Rename BUILD files
+    ctx.file("rename_build_files.sh", "find ./test -name \"BUILD\" -exec sh -c 'mv \"$1\" \"$(dirname $1)/test.BUILD\"' _ {} \;", True)
+    result = ctx.execute(["./rename_build_files.sh"])
+    _print_exec_results("Rename BUILD files in test output", result, True, args)
+
+    # create a root BUILD file with a filegroup
+    ctx.file("test/BUILD", """package(default_visibility = ["//visibility:public"])
+exports_files(["empty"])
+filegroup(
+    name = "exported_testdata",
+    srcs = glob(["**/*"]),
+)
+""", False)
+
+    # Create an empty file to reference in test.
```

Comment:
Technically, I don't think this is necessary because doing "$(location @rbe_autoconf_generate_no_cc//test:exported_testdata)" should work as well. When the file group has more than one file, this should just randomly pick one file from that group. Since you are only using this to obtain the root directory, that should serve your purpose.

Reply:
I dont want to run into issues with the functionality of $(location ...) for a filegroup, its more reliable to just create the file.